roxie/ccd/ccdlistener.cpp

```diff
@@ -943,6 +943,27 @@ extern void updateAffinity(unsigned __int64 affinity)
     RoxieListener::updateAffinity();
 }
 
+//--------------------------------------------------------------------------------------------------------------------
+
+void ContextLogger::exportStatsToSpan(bool failed, unsigned elapsed, unsigned memused, unsigned agentsDuplicates, unsigned agentsResends)
+{
+    if (activeSpan->isRecording())
+    {
+        setSpanAttribute("time_elapsed", elapsed*1000000ULL);
```

Comment:
I forgot to mention in our offline conversation, if some of these stats pertain to the entire roxie query (as opposed to a span w/in the query) then the names should reflect that.
"query.time_elapsed", or "query_time_elapsed", etc.